src/css/components/_textarea.scss

```diff
@@ -0,0 +1,15 @@
+.text-wrapper {
+  width: 840px;
+  margin: auto;
+}
+
+.textarea {
+  width: 820px;
+  padding: 10px;
+  font-family: $main-font;
+  font-size: 20px;
+  background-color: $main-color;
+  border-radius: 10px;
+  border: 2px solid rgb(141, 134, 134);
```

Comment:
Please move border color to your colors variables.

Reply:
done